### PR TITLE
Fix help urls

### PR DESCRIPTION
**Pull request by pablomarga**
Hi teams this PR solves:
- "help" references to` /current/` instead of the app version. 

Related Issue #953

### CODE REVIEW

File: SplunkAppForWazuh/appserver/static/js/run/run.js [hunk 1]
```diff
@@ -6,18 +6,35 @@ define(['./module'], function(module) {
...
       //Go to last state or to a specified tab if "currentTab" param is specified in the url
       $navigationService.manageState()
-
+      async function getAppVersion(){
+        try {
+          const result = await $requestService.httpReq(
+            'GET',
+            '/manager/app_info'
+          )
+          console.log('result',result)
```
Comment (Desvelao) [inline SplunkAppForWazuh/appserver/static/js/run/run.js:30]: Remove this `console.log`

File: SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ) {
+        let appInfo = {revision:'',version:'',splunk_version:''};
+        let documentationAppVersion='';
+        const getAppVersion = () => {
+            return appInfo;
+        }
+
+        /**
+         * Generates and returns the browser base URL + Splunk Port
+         */
```
Comment (Desvelao) [inline SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js:14]: Add the correct documentation to the function. That description doesn't apply to the function

File: SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            documentationAppVersion = [major, minor].join('.'); 
+        }
+        
+        const getDocumentacionVersion = () => {
+            return documentationAppVersion;
+        }
+
+        const getDocuUrl = (url) => {
+            return url.replace('/current/',`/${documentationAppVersion}/`)
+        }
```
Comment (Desvelao) [inline SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js:27]: Remove it if this is not used

File: SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        /**
+         * Set the info about the app and splunk
+         */
+        const setAppInfo = (info) => {
+            let appInfo = info;
+            const [major, minor] = appInfo.version.split('.');
+            documentationAppVersion = [major, minor].join('.');
+        }
+
+        const getDocumentacionVersion = () => {
```
Comment (frankeros) [inline SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js:21]: Fix spelling

File: SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            return documentationAppVersion;
+        }
+
+        const service = {
+            getAppVersion: getAppVersion,
+            setAppInfo: setAppInfo,
+            getDocumentacionVersion: getDocumentacionVersion
```
Comment (frankeros) [inline SplunkAppForWazuh/appserver/static/js/services/app-version/appVersionService.js:28]: same here

**Review by frankeros — CHANGES_REQUESTED** [review]

**Pull request merged**